sys/shell/shell.c

```diff
@@ -308,4 +316,8 @@ void shell_run_once(const shell_command_t *shell_commands,
 
         print_prompt();
     }
+
+#ifdef SHELL_SHUTDOWN_ON_EOF
```

Comment:
```suggestion
#if SHELL_SHUTDOWN_ON_EOF
```

That way it can be unset on native.

Reply:
This gives

```C
/home/benpicco/dev/RIOT-BHT/sys/shell/shell.c: In function ‘shell_run_once’:
/home/benpicco/dev/RIOT-BHT/sys/shell/shell.c:320:26: error: #if with no expression
  320 | #if SHELL_SHUTDOWN_ON_EOF
      | 
```

but looks like that is why we have that new `IS_ACTIVE()` macro now.